test/fixtures/cookbooks/test/metadata.rb

```diff
@@ -0,0 +1,8 @@
+name             'test'
```

Comment:
For semantics sake, it might be best to name this `pdns_test` so it's a bit clearer, but it's not a major issue if it's not. I also tend to put cookbooks in just `test/cookbooks` instead of the additional fixtures directory since that is typically the only 'fixture' other than the other items that have their own directory like data bags, nodes, environments, etc. (all under integration).